copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -57,3 +57,24 @@ Turbine Engine Start
 ====================
 
 A special RC Auxiliary Function ("159") is implemented to provide the start signal RC switch for turbine engines. When armed and RSC is idle, the high position signals the helicopter rotor speed governor to ramp the throttle to full and back to idle, which signals the turbine engine ECU to initiate the start sequence. The switch must be set back low and aircraft has to be disarmed to re-enable this feature.
+
+Practice (Manual) Autorotation Setup
+====================================
+
+In versions 4.4 and later, the ability to conduct power re-engagement from an autorotation has been added.  This feature will work with ESC's using their built-in governor, the ardupilot throttle curve or the ardupilot built-in governor.  Three parameters were added to accommodate this feature.
+
+* :ref:`H_RSC_AROT_MN_EN <H_RSC_AROT_MN_EN>` - Enables manual autorotations (0 - Disable, 1 - Enable)
+* :ref:`H_RSC_AROT_ENG_T <H_RSC_AROT_ENG_T>` - Time to ramp throttle back to motorlock enabled value during power recovery in seconds
+* :ref:`H_RSC_AROT_IDLE <H_RSC_AROT_IDLE>` - Idle throttle value in percent used during autorotation.  This can be set to the value that signals the ESC to perform a fast spool-up when performing a power recovery.
+
+ESC built-in Governor
```

Comment:
we should call this ESC Governor    you use "built-in" lower down and its confusing